src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -119,14 +123,16 @@ public static string RemoveVersions(string source)
         }
 
         /// <summary>
-        /// This returns a <see cref="Matcher"/> that can be used to match on a path whose versions have been removed via
-        /// <see cref="RemoveVersions(string)"/>.
+        /// This returns a <see cref="Matcher"/> that can be used to match on a path whose versions and RID have been removed via
+        /// <see cref="RemoveVersions(string)"/> and <see cref="RemoveRids(string, bool)"/>
         /// </summary>
         public static Matcher GetFileMatcherFromPath(string path)
         {
             path = path
                 .Replace(SemanticVersionPlaceholder, SemanticVersionPlaceholderMatchingPattern)
-                .Replace(NonSemanticVersionPlaceholder, NonSemanticVersionPlaceholderMatchingPattern);
+                .Replace(NonSemanticVersionPlaceholder, NonSemanticVersionPlaceholderMatchingPattern)
+                .Replace(TargetRidPlaceholder, TargetRidPlaceholderMatchingPattern)
+                .Replace(PortableRidPlaceholder, PortableRidPlaceholderMatchingPattern);
```

Comment:
Given the Target and Portable rid placeholders are the same, how would the portable replace ever be needed?

Reply:
The Target rid placeholder is `banana-rid` and the Portable rid placeholder is `portable-rid`. They're matching pattern placeholders are both `*-*` though.

Reply:
I transposed the arguments in my mind.  I see that it is doing now.